main/src/library/pkger/pkger.flix

```diff
@@ -0,0 +1,43 @@
+pub enum SemVerComponent {
+    case Integer(Int),
+    case Wildcard,
+    case NotPresent
```

Comment:
Do we want to allow all of these? Are all combinations valid, e.g. a semver that is: 42.NotPresent._ ?

Reply:
I thought about that while writing, and came to the conclusion that it was the wrong way to represent SemVer. But I also wanted something down to represent SemVer in some way, so I kept it as a temporary thing. Still not sure on what a better canonical way to represent SemVer is. It should be something that, as much as possible, disallows invalid specifications like the one you gave.